tests/core/job/test_job_manager.py

```diff
@@ -60,9 +60,10 @@ def lock_multiply(lock, nb1: float, nb2: float):
 
 def test_create_jobs():
     Config.configure_job_executions(mode=JobConfig._DEVELOPMENT_MODE)
-    _SchedulerFactory._build_dispatcher()
     task = _create_task(multiply, name="get_job")
 
+    _SchedulerFactory._build_dispatcher()
```

Comment:
@trgiangdo 
why do we need to move the build dispatcher call here? It doesn't seem like it makes any difference right? If it doesn't make any difference then I have no problem with it, if it does, can you briefly explain it to me?

Reply:
Same with many other places apparently

Reply:
The use-case for block update config, for now, is to block all configuration update after running the core service (or submit a task), which equivalent to start a JobDispatcher.

That's the reason behind moving all `_build_dispatcher()` after all of the configuration.

Reply:
Ah my bad, I forgot the create_task function has some config in it, nvm :D